src/__tests__/components.test.js

```diff
@@ -0,0 +1,86 @@
+import { render } from '@testing-library/react';
+import { StrictMode } from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import About from '../pages/About';
+import Characters from '../pages/Characters';
+import store from '../redux/configureStore';
+import '@testing-library/jest-dom';
+import NavBar from '../components/NavBar';
+import Footer from '../components/Footer';
+import Hero from '../components/Hero';
+
+describe('Testing About page', () => {
+  it('Should match the snapshot', () => {
+    const about = render(
+      <StrictMode>
+        <Router>
+          <Provider store={store}>
+            <About />
+          </Provider>
+        </Router>
+      </StrictMode>,
+    );
```

Comment:
- [ ] As requested in project requirements, let's add tests for some pure functions in Redux code.